### ISSUE TITLE
"Phone-home" feature is opt-out

### ISSUE THREAD

**Issue by maxidor** [matrix-org/package-synapse-debian#26]
It should be opt-in for privacy reason.
See https://matrix.to/#/!XqBunHwQIXUiqCaoxq:matrix.org/$1523296112136GHSqp:kamax.io

**Comment by uhoreg** [comment]
FWIW, until this is fixed, you should be able to disable it for non-interactive installs by running the following *before* apt-installing the matrix-synapse package:

```
echo "matrix-synapse matrix-synapse/report-stats boolean false" | sudo debconf-set-selections
```

(untested, but *should* work)

**Comment by erikjohnston** [comment]
Thanks! Hopefully this should be fixed in fc0981b14d8ae9bc048689f4edde04fd45a9f827
